.github/workflows/release.yml

```diff
@@ -280,10 +287,16 @@ jobs:
 
           $(go env GOPATH)/bin/ziti-ci configure-git
 
+          # workaround: prune the boring builds from release tarballs so it will not force the regular ziti binary into a nested path /ziti/ziti
```

Comment:
@plorenz I hesitate to change how ziti-ci archives artifacts: nest in `cmd.name` directory if multiple files, else top-level. I'm unsure what side effects changing that might have, and I remember it was problematic when we triggered that behavior by unintentionally including a helper binary in the Linux release archive.